app/controllers/hub/faq_categories_controller.rb

```diff
@@ -48,7 +48,6 @@ def destroy
       begin
         ActiveRecord::Base.transaction do
           # if faq_items deletion fails, don't destroy their faq_category
-          @faq_category.faq_items.map(&:destroy)
```

Comment:
since dependent destroy should handle things for us, I'm removing this-- this also resolves some spec failures about paper trail, but not sure why that's happening (yet)